src/api/focus/Focus.ts

```diff
@@ -285,38 +303,38 @@ class Focus {
     });
   }
 
-  async command(cmd, ...args) {
-    if (typeof this.commands[cmd] === "function") {
-      return this.commands[cmd](this, ...args);
+  async command(cmd: string, ...args: unknown[]) {
+    const override = this.commands[cmd];
+    if (typeof override === "function") {
+      return override(this, ...args);
     }
-    if (typeof this.commands[cmd] === "object") {
-      return this.commands[cmd].focus(this, ...args);
+    if (typeof override === "object") {
+      return override.focus(this, ...args);
     }
     return this.request(cmd, ...args);
   }
 
-  addCommands(cmds) {
+  addCommands(cmds: CommandOverrides) {
     Object.assign(this.commands, cmds);
   }
 
-  addMethod(methodName, command) {
-    if (this[methodName]) {
-      const tmp = this[methodName];
-      this[methodName] = (...args) => {
+  addMethod(methodName: string, command: string) {
```

Comment:
This method while being used in two places,
the added methods are not being used.

I think this can be removed.